projects/vdk-plugins/vdk-dag/src/vdk/plugin/dag/dag.py

```diff
@@ -47,6 +53,25 @@ def __init__(self, team_name: str, dags_config: DagPluginConfiguration):
             time_between_status_check_seconds=dags_config.dags_time_between_status_check_seconds(),
         )
         self._dag_validator = DagValidator()
+        if job_name is not None and execution_id is not None:
+            try:
+                self._started_by = (
+                    self._job_executor.execution_type(job_name, team_name, execution_id)
+                    + "/"
+                    + job_name
+                )
+            except ApiException as e:
+                if e.status == 404:
+                    log.debug(
+                        f"Local job runs return 404 status when getting the execution type: {e}"
+                    )
+                else:
+                    log.info(
+                        f"Unexpected error while checking for job execution type: {e}"
```

Comment:
same comment as above. Review other logs and make sure there is context that may be helpful when root causing an issue.

Reply:
Done.